modules/serverless-kn-func-invoke-reference.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * serverless/reference/kn-func-ref.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-invoke-reference_{context}"]
+= kn func invoke optional parameters
+
+You can specify optional parameters for the sent request by using the `kn func invoke` CLI command flags.
+
+* `-t`, `--target`: Specifies the target instance of the invoked function, for example, `local` or `remote`.
+* `-f`, `--format`: Specifies the format of the message, for example, `CloudEvent` or `http`.
+* `--id`: A unique string identifier for the request.
+* `-n`, `--namespace`: The namespace on the cluster.
+* `--source`: Sender name for the request.
+* `--type`: Type of the request, for example, `boson.fn`.
```

Comment:
Similar to above, it might be beneficial to note that this derives from a CloudEvent attribute.
```suggestion
* `--type`: Type of the request, for example, `boson.fn`. This corresponds to the CloudEvent `type` attribute.
```